App.tsx

```diff
@@ -46,20 +50,33 @@ const Tabs = () => {
   );
 };
 
-const App = () => {
+const ModalNav = () => {
   const Tab = createNativeStackNavigator();
 
   return (
-    <NavigationContainer>
-      <Tab.Navigator>
-        <Tab.Screen
-          name="Tabs"
-          component={Tabs}
-          options={{ presentation: "modal", headerShown: false }}
-        />
-      </Tab.Navigator>
-    </NavigationContainer>
+    <StoreProvider>
+      <NavigationContainer>
+        <Tab.Navigator>
+          <Tab.Screen
+            name="Tabs"
+            component={Tabs}
+            options={{ presentation: "modal", headerShown: false }}
+          />
+          <Tab.Screen
+            name="AddReview"
+            component={AddReview}
+            options={{ presentation: "modal", headerShown: false }}
+          />
+        </Tab.Navigator>
+      </NavigationContainer>
+    </StoreProvider>
   );
 };
 
+const App = () => {
+  useEffect(() => {
+    SplashScreen.hide();
+  }, []);
+  return <ModalNav />;
+};
```

Comment:
I believe you can put the `ModalNav` template within this `App` function component!